dashboard/modules/metrics/dashboards/data_dashboard_panels.py

```diff
@@ -0,0 +1,94 @@
+# flake8: noqa E501
+
+from ray.dashboard.modules.metrics.dashboards.common import (
+    DashboardConfig,
+    Panel,
+    Target,
+)
+
+DATA_GRAFANA_PANELS = [
+    Panel(
+        id=45,
+        title="Bytes Spilled",
+        description="Amount spilled by dataset operators.",
+        unit="bytes",
+        targets=[
+            Target(
+                expr="sum(ray_data_spilled_bytes{{{global_filters}}}) by (dataset)",
+                legend="Bytes Spilled: {{dataset}}",
+            )
+        ],
+    ),
+    Panel(
+        id=46,
+        title="Bytes Allocated",
+        description="Amount allocated by dataset operators.",
+        unit="bytes",
+        targets=[
+            Target(
+                expr="sum(ray_data_allocated_bytes{{{global_filters}}}) by (dataset)",
+                legend="Bytes Allocated: {{dataset}}",
+            )
+        ],
+    ),
+    Panel(
+        id=47,
+        title="Bytes Freed",
+        description="Amount freed by dataset operators.",
+        unit="bytes",
+        targets=[
+            Target(
+                expr="sum(ray_data_freed_bytes{{{global_filters}}}) by (dataset)",
```

Comment:
for each of these panels, you'll need to add a variable for DatasetId. Since we want to add a filter in the UI and the grafana dashboard.

Reply:
You'll also need to add the variables to the list in the "templating" object in the `default_grafana_dashboard_base.json` file